database/engine/README.md

```diff
@@ -6,17 +6,88 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/engine/
 
 # Database engine
```

Comment:
I think it's time for this whole file to come into the database/README.md
I don't see why it need to be a separate file. 
For sure it will make the database/README.md much larger, but I don't see any benefits from having two separate files. dbengine IS our database, the others are side notes for very specific use cases